.github/workflows/test-pr.yml

```diff
@@ -10,6 +10,11 @@ defaults:
 env:
   # https://github.com/docker-library/bashbrew/issues/10
   GIT_LFS_SKIP_SMUDGE: 1
+  
+# Cancel existing runs if user makes another push
+concurrency:
+  group: "${{ github.ref }}"
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```

Comment:
Just want to note since it's ambiguous from the docs, `github.ref` is going to be of the format `refs/pull/:prNumber/merge` (not `refs/heads/<branch_name>` as the https://docs.github.com/en/actions/learn-github-actions/contexts#github-context docs imply -> https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request).

Just a minor suggestion to adjust for consistency:

```suggestion
# cancel existing runs if user makes another push
concurrency:
  group: ${{ github.ref }}
  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```